backend/inspirehep/submissions/views.py

```diff
@@ -259,12 +260,23 @@ def create_ticket(self, record, rt_template):
         )
 
 
+class ExperimentSubmissionsResource(BaseSubmissionsResource):
+    def post(self):
+        """Adds new experiment record"""
+
+        data = self.load_data_from_request()["data"]
```

Comment:
what if there'll be key error?

I would suggest to create a dict with required fields and then update it with data from request.